frontend/apps/gate/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
```

Comment:
Shall we have a heading as well

Reply:
Do we need this readme file?

Reply:
Better we keep. I have changed the content.